spectacles/printer.py

```diff
@@ -79,9 +79,19 @@ def print_content_error(
 
 
 def print_data_test_error(
-    model: str, explore: str, test_name: str, message: str, lookml_url: str
+    model: Optional[str],
+    explore: Optional[str],
+    test_name: Optional[str],
```

Comment:
What are the cases where these aren't defined?

Reply:
This seems to happen when there's a SQL error in a data test's explore.

Reply:
Not in the case I was seeing! But maybe something was failing further upstream?